src/components/base/tooltip/Tooltip.js

```diff
@@ -14,19 +15,33 @@ export default function Tooltip({
   trigger = ["focus"],
   iconSize = 3,
   children,
+  show,
+  tooltipRef,
+  toolTipClassName,
 }) {
   const spanRef = useRef();
+
+  const [mounted, setMounted] = useState(false);
+
+  //fixes hydration error. https://stackoverflow.com/questions/75094010/nextjs-13-hydration-failed-because-the-initial-ui-does-not-match-what-was-render
+  useEffect(() => {
+    setMounted(true);
+  }, []);
+
+  if (!mounted) return null;
```

Comment:
Jeg tror faktisk bare at du kan tjekke om children er defineret, og så slippe for både `useEffect` og `useState`

Reply:
Det virkede desværre ikke 🥲